microsetta_interface/implementation.py

```diff
@@ -1155,6 +1163,21 @@ def post_remove_source(*,
     return _refresh_state_and_route_to_sink(account_id)
 
 
+# same as above.
```

Comment:
Can you clarify this comment in case someone eventually sticks a function between post_remove_source() and post_request_account_removal()?